public/js/custom.js

```diff
@@ -4,3 +4,15 @@ $(".like-button").on('click', function() {
     $(".like-button").text('LIKE COUNT: ' + JSON.parse(data).likes);
   });
 });
+
+$(".in-application-search input").on('keyup', function() {
+  var query = $(this).val();
+  $.get('/api/search/' + query, function(data) {
+    $(".search-results").html('');
+    data.forEach(function(element) {
+      $(".search-results").append(
+        $('<li>' + element.firstname + ' ' + element.lastname + '</li>')
+      );
+    });
+  });
+});
```

Comment:
we access the data, the response body of the server. Check server route handler to see how I did that. Here we are just iterating the response